### ISSUE TITLE
crash after return to title

### ISSUE THREAD

**Issue by newwby** [newwby/mn_missilemania_idv2#9]
crash to desktop occurs after playing at least one game instance and returning to title

**Comment by newwby** [comment]
appears to be related to texture loading issue on main repo
objectDB leak bug resurfacing, run w/ '--verbose' runtime arg and reference main repo issue regarding this

**Comment by newwby** [comment]
under debug+verbose managed to reproduce the ctd only after quitting to title, _not_ after death

**log for ctd on quit to title**
[22-12-07-eve_debug-log_is9_crash_on_quit_to_title.txt](https://github.com/newwby/mn_missilemania_idv2/files/10180217/22-12-07-eve_debug-log_is9_crash_on_quit_to_title.txt)

see next comment for log for title after death and restarting game

**Comment by newwby** [comment]
[separating to own comment to raise new issue]

under debug+verbose managed to encounter a new issue (potentially caused by losing reference to globalPool.main_player?)

**error pictured here**
![image](https://user-images.githubusercontent.com/60653155/206303260-f6fd4820-7f98-4a9c-9024-b3308e98b786.png)

as shown above, you can see the off-screen indicators are leaving the space at the screen margin where they are supposed to be clamped, and moving toward the player. This is however a symptom of the actual problem (first instinct was that the enemies were somehow rendered invisible but this proved not be true) - the enemy tracking is absolutely borked. Enemies are behaving as though a random point in the game space is the player, and acting accordingly. Indicators are hidden in-screen by the screen visibility trackers for enemies, so if enemies aren't actually entering the player viewport they won't be triggering the indicators hide() method. Unsure of the issue on main repo, but believe this has happened before when player targeting reference got unset on globalPool.

**log for title after death and restarting game**
[22-12-07-eve_debug_log_is9_peculiar_behaviour_after_death.txt](https://github.com/newwby/mn_missilemania_idv2/files/10180263/22-12-07-eve_debug_log_is9_peculiar_behaviour_after_death.txt)

will update this as new issue

**Comment by newwby** [comment]
resolved as of 19a28f52c9401d05b60328be6072b4f383903997

**Comment by newwby** [comment]
mistakenly closed (was on wrong tab and didn't look before commenting/closing)
this bug is still open but a fix is underway
